### ISSUE TITLE
Issued currency payment on Testnet

### ISSUE THREAD

**Issue by AmitMirgal** [XRPLF/xrpl.js#1908]
I was able to generate the wallet and set the trust line between the issuer and the newly generated wallet.
Once that is done successfully I am not able to send the issued currency between accounts...
Receiving the following error
```
XrplError: The latest ledger sequence 24820660 is greater than the transaction's LastLedgerSequence (24820660).
```

```javascript
        const client = new xrpl.Client("wss://s.altnet.rippletest.net:51233");
	await client.connect();

	const wallet = xrpl.Wallet.fromSeed("sEdVBbmM9U1keWJATkhVpRUFT3V8FLE");

	// * send IOU currency
	const preparedTransaction = await client.autofill({
		TransactionType: "Payment",
		Account: "r4YUpzoMuRKSp5uRqbDGDLb5hq21j9xZyn",
		Destination: wallet.classicAddress,
		Amount: {
			currency: "EAT",
			value: "20",
			issuer: "r4YUpzoMuRKSp5uRqbDGDLb5hq21j9xZyn",
		},
		Fee: "12",
	});

	// * Sign prepared instructions
        const signed = wallet.sign(preparedTransaction);

	// * Submit signed blob
	await client.submitAndWait(signed.tx_blob);

	// Disconnect when done (If you omit this, Node.js won't end the process)
	await client.disconnect();
```

could anyone please help me.

**Comment by mvadari** [comment]
Your transaction is failing - likely because you're trying to sign a transaction that isn't being sent from your account. Right now, you're trying to send from Account A (`r4YUpzoMuRKSp5uRqbDGDLb5hq21j9xZyn`) to Account B (`wallet.classic_address`), but you're trying to sign the transaction with Account B. Account B presumably doesn't have authority to send value from Account A - you have to have Account A sign the transaction for this to go through.

You're running into a minor bug that is fixed in v2.1.1 - where the error message isn't properly printed out.

**Comment by AmitMirgal** [comment]
@mvadari thanks for your super quick response. it's working as expected now... Appreciate it...
